src/main/java/seedu/address/logic/commands/edit/EditModuleRoleOperation.java

```diff
@@ -33,4 +37,41 @@ public abstract class EditModuleRoleOperation {
     public static boolean isValidModuleRoleOperation(String operation) {
         return operation.toLowerCase().matches(VALIDATION_REGEX);
     }
+
+    /**
+     * Returns a description of the change in module roles.
+     * @param moduleRoleMapBefore The module role map before the change.
+     * @param moduleRoleMapAfter The module role map after the change.
+     * @return A description of the change in module roles.
```

Comment:
I think can include an example over here for easier visualization, you can refer to `getModuleCodeString` method Javadoc to see how it explains what it does

Reply:
good suggestion, added!